vars/README.md

```diff
@@ -0,0 +1,13 @@
+# Top Level Variables
+
+List of top level variables.
+
+### Variables
+| Variable        | Default Value           | Description  |
+|:------------- |:-------------|:----- |
+| cluster | openshift | The cluster we're running on |
+| namespace | kube-system | Namespace to create resources |
+| openshift_version | 3.7 | OpenShift cluster version. Either 3.7 or 3.9 |
+| manifest_version | release | KubeVirt manifest version |
+| docker_tag | latest | Container image tag |
+| storage_role | None | Storage role to install with KubeVirt |
```

Comment:
I am little bit confused by name of this variable, I am getting feeling that I should provide name of `storage_role` to be used, and according to playbook I see that `storage.yml` playbook is loaded for any value != None.
I would either improve doc for this variable, or change name.